### ISSUE TITLE
Clone without -r option is not working

### ISSUE THREAD

**Issue by abishai** [pr1ntf/iohyve#213]
```
iohyve v0.7.7 2016/11/10 I Think I'll Go for a Walk Edition
```
I can't clone virtual box without supplying -r option.
```
abishai@vhost:~ % doas iohyve clone quik-dev.foo.bar quik-test.foo.bar
Not a valid guest name
```
The same command with -r option works.

**Comment by pr1ntf** [comment]
This indeed is broken right now, and has been for some time. There is currently no way of making an exact clone in `iohyve` at this time. 

We will probably take out support for exact clones and just make `-r` the default behaviour. We will keep this issue open as a reminder, or until we write this down elsewhere (like the [roadmap](https://github.com/pr1ntf/iohyve/wiki/General-Path-to-1.0)).

**Comment by abishai** [comment]
Maybe add the ability to provide custom tap and console options when using -r as it autogenerates them, which is not always desirable?

**Comment by SpaIns** [comment]
I'm currently on the latest stable version,  FreeNAS-9.10.1-U4.  I know there were talks in the forums about trying to upgrade iohyve without freenas devs having to push through the updates, but I don't think anything came of it.  I also saw a post saying they weren't sure if it was going to make it into 10.0, but I'll try to figure out where the best place is to make a feature request, because I have really enjoyed using iohyve up to this point.

**Comment by mariusvw** [comment]
Can anyone test if this is resolved with the changes in issue #215, thank you! :)

**Comment by pr1ntf** [comment]
Worked just fine for me. Thank you everyone for your help!
